specs/paymentrequest.html

```diff
@@ -646,64 +709,64 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
     </section>
 
     <section>
-      <h2>PaymentDetails dictionary</h2>
+      <h2>PaymentMethod</h2>
       <pre class="idl">
-dictionary PaymentDetails {
-  sequence&lt;PaymentItem&gt; items;
-  sequence&lt;ShippingOption&gt; shippingOptions;
+dictionary PaymentMethod {
+  required sequence&lt;DOMString&gt; identifiers;
+  object data;
 };
```

Comment:
I'm very wary of catch-all containers such as "data". I'd rather we figure out how we do extensibility properly before punting to "generic data buckets". So, -1 to this line. The rest of the PR is looking good - still reviewing it.